### PR TITLE
remove controller

### PR DESCRIPTION
**Pull request by sdelamo**
This PR removes the controller and instructs users to expose the JSON Schema via static resources.

### CODE REVIEW

File: test-suite-kotlin/src/test/groovy/io/micronaut/jsonschema/test/ServingSchemasSpec.groovy [hunk 1]
```diff
@@ -33,8 +32,7 @@ class ServingSchemasSpec extends Specification {
 
         when:
         result = client.toBlocking().retrieve(
-                HttpRequest.GET(server.getURI().resolve("/schemas/red-winged-blackbird")).accept(MediaType.APPLICATION_JSON),
-                String.class
+                HttpRequest.GET(server.getURI().resolve("/schemas/red-winged-blackbird.schema.json"))
```
Comment (andriy-dmytruk) [inline test-suite-kotlin/src/test/groovy/io/micronaut/jsonschema/test/ServingSchemasSpec.groovy:35]: Why was media type removed?

**Comment by andriy-dmytruk** [comment]
Maybe we want to supply to correct media type. But perhaps we also want it to be `application/schema+json`.

It is indeed better to remove the controllers, so I will merge and we can think of this later.

**Comment by sdelamo** [comment]
> Maybe we want to supply to correct media type. But perhaps we also want it to be `application/schema+json`.
> 
> It is indeed better to remove the controllers, so I will merge and we can think of this later.

Yes, I will add a PR to Core to support the `application/schema+json` media type.

File: test-suite-kotlin/src/test/groovy/io/micronaut/jsonschema/test/ServingSchemasSpec.groovy (hunk 1 above)
Comment (sdelamo) [reply test-suite-kotlin/src/test/groovy/io/micronaut/jsonschema/test/ServingSchemasSpec.groovy:35]: it is unnecessary.

**Pull request merged**